compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -974,7 +982,10 @@ protected Result seekMatch(ParserRuleContext currentCtx, int lookahead, int curr
                     hasMatch = nextToken.kind == SyntaxKind.ASCENDING_KEYWORD ||
                             nextToken.kind == SyntaxKind.DESCENDING_KEYWORD;
                     break;
-
+                case PUBLIC_OR_PRIVATE_KEYWORD:
```

Comment:
Would "object member qualifier" be a better name?

Reply:
will change it